CONTRIBUTING.md

```diff
@@ -211,3 +211,42 @@ command above.
   test outputs are in one file.
   However, it may slow down the tests for not running in parallel
   and may cause the test to timeout.
+
+## Contributing to Keras applications
+
+Contributions to the
+[pre-trained application library](https://keras.io/api/applications/)
+are welcome. Code for Keras applications is located in Keras repository in
+[keras/applications](https://github.com/keras-team/keras/blob/master/keras/applications).
+When contributing to Keras applications, please keep following checklist in
+mind.
+
+ - Keras applications must implement an established and widely used model.
```

Comment:
@mattdangerw 
How is the term **...widely used model** defined? Is it based on only the number of citations of the model used in well-reputed conferences/journals? In that case, I think the conf/journal publisher should be realized also. 20 citations from random journals/publishers would be easy to find these days.

Reply:
Yeah, @fchollet could probably add more color. I don't think there is any fool proof way of defining widely used. 20 citations is just a minimum bar really. We can't enumerate all reputable journals.

Reply:
We don't want to set too high a bar -- 20 citations is enough to filter out the things we want to exclude. It's really more about excluding the clutter than about formally defining "widely used".